src/main/java/org/broadinstitute/hellbender/tools/walkers/featuremapping/FlowFeatureMapperArgumentCollection.java

```diff
@@ -89,6 +89,18 @@ enum MappingFeatureEnum {
     @Argument(fullName = "keep-supplementary-alignments", doc = "keep supplementary alignments ?", optional = true)
     public boolean keepSupplementaryAlignments = false;
 
+    /**
+     *  do not emit edit distance at all
+     **/
+    @Argument(fullName = "no-edit-distance", doc = "do not emit edit distance at all ?", optional = true)
+    public boolean noEditDistance = false;
+
+    /**
```

Comment:
Can we switch the nm-edit-distance to the default?

Reply:
NM now default